libs/hdf-converters/src/neuvector-mapper.ts

```diff
@@ -0,0 +1,311 @@
+  return regex.exec(name)?.[0];
+}
+
+function rhsaTag(name: string): string | undefined {
+  const regex = /RHSA-[a-z|0-9]{4}:[a-z|0-9]{4}/;
+  return regex.exec(name)?.[0];
+}
+
+function cveIdMatches(cveName: string): (value: RESTModuleCve) => boolean {
+  return (cve: RESTModuleCve) => cve.name === cveName;
+}
+
+function universalTags(): MappedTransform<
+  {[key: string]: any} & ILookupPath,
+  ILookupPath
+> {
+  // Heimdall currently doesn't have a way to display passthrough data, and this information would be useful to view per vulnerability.
+  return {
+    envs: {
+      path: '$.report.envs',
+      transformer: (envs: string[]) => (!envs ? undefined : envs)
+    },
+    cmds: {
+      path: '$.report.cmds',
+      transformer: (cmds: string[]) => (!cmds ? undefined : cmds)
+    }
+  };
+}
+
+export class NeuvectorMapper extends BaseConverter {
+  withRaw: boolean;
+  rawData: NeuvectorScanJson;
+  getModule: (moduleName: string) => RESTScanModule | undefined;
+
+  memoizedGetModule(): (moduleName: string) => RESTScanModule | undefined {
+    return (moduleName: string) => {
+      return this.rawData.report.modules?.find(
+        (value: RESTScanModule) => value.name === moduleName
+      );
+    };
+  }
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: null
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null
+    },
+    profiles: [
+      {
+        name: 'Neuvector',
+        title: null,
+        version: null,
+        maintainer: null,
+        summary: null,
+        license: null,
+        copyright: null,
+        copyright_email: null,
+        supports: [],
+        attributes: [],
+        depends: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'report.vulnerabilities',
+            key: 'id',
+            tags: {
+              cves: {path: 'cves'},
+              cpes: {
+                path: 'cpes',
+                transformer: (cpes: string[]) => (!cpes ? undefined : cpes)
+              },
+              cwe: {
+                path: 'description',
+                transformer: cweTags
+              },
+              nist: {
+                path: 'description',
+                transformer: (description: string) =>
+                  nistTags(cweTags(description))
+              },
+              ghsa: {path: 'name', transformer: ghsaTag},
+              rhsa: {path: 'name', transformer: rhsaTag},
+              // `score` is confirmed to be CVSS v2 in https://github.com/neuvector/scanner/blob/765fb1db2cf678ea6c6d386f3eb0f720311d745a/cvetools/cvesearch.go#L1416
+              score: {path: 'score'},
+              vectors: {path: 'vectors'},
+              // In the neuvector/scanner dashboard, the CVSS v2 and v3 scores are selectable by a dropdown.
```

Comment:
unless you were able to spin up the application, we didn't confirm this.  it was just our conjecture.